modules/fswatch/fswatch_profile_linux.go

```diff
@@ -0,0 +1,22 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Contributor: Aaron Meihm ameihm@mozilla.com [:alm]
+
+package fswatch /* import "mig.ninja/mig/modules/fswatch" */
+
+var localFsWatchProfile = fsWatchProfile{
+	[]fsWatchProfileEntry{
+		{"/boot", []fsWatchObject{}},
+		{"/etc/cron.d", []fsWatchObject{}},
+		{"/var/spool/cron", []fsWatchObject{}},
+		{"/bin", []fsWatchObject{}},
+		{"/sbin", []fsWatchObject{}},
+		{"/usr/bin", []fsWatchObject{}},
+		{"/usr/sbin", []fsWatchObject{}},
+		{"/etc", []fsWatchObject{}},
+		{"/etc/init.d", []fsWatchObject{}},
+		{"/etc/systemd/system", []fsWatchObject{}},
+	},
+}
```

Comment:
That's very broad for a default profile. I'd prefer to be a lot more specific and give administrators the choice of going with a broad coverage, to limit the noise when enabling this with a default conf.